Assets/HoloToolkit/Build/Editor/BuildSLNUtilities.cs

```diff
@@ -102,29 +102,22 @@ public static void PerformBuild(BuildInfo buildInfo)
 
             EditorUserBuildSettings.SwitchActiveBuildTarget(buildTargetGroup, buildInfo.BuildTarget);
 
-            WSASDK oldWSASDK = EditorUserBuildSettings.wsaSDK;
-            if (buildInfo.WSASdk.HasValue)
-            {
-                EditorUserBuildSettings.wsaSDK = buildInfo.WSASdk.Value;
-            }
+            WSAUWPBuildType? oldWSAUWPBuildType = EditorUserBuildSettings.wsaUWPBuildType;
 
-            WSAUWPBuildType? oldWSAUWPBuildType = null;
-            if (EditorUserBuildSettings.wsaSDK == WSASDK.UWP)
+            if (buildInfo.WSAUWPBuildType.HasValue)
             {
-                oldWSAUWPBuildType = EditorUserBuildSettings.wsaUWPBuildType;
-                if (buildInfo.WSAUWPBuildType.HasValue)
-                {
-                    EditorUserBuildSettings.wsaUWPBuildType = buildInfo.WSAUWPBuildType.Value;
-                }
+                EditorUserBuildSettings.wsaUWPBuildType = buildInfo.WSAUWPBuildType.Value;
```

Comment:
Your PR description says "Removed references to `EditorUserBuildSettings.wsaUWPBuildType`. It will be removed in future versions of Unity." It's still here though. Did you mean `EditorUserBuildSettings.wsaSDK`?